### PR TITLE
Fluffy: Make max number of nodes to gossip content to configurable

### PR DESCRIPTION
**Pull request by web3-developer**

### CODE REVIEW

File: fluffy/network/wire/portal_protocol.nim [hunk 1]
```diff
@@ -1518,17 +1519,17 @@ proc neighborhoodGossip*(
         elif node.id != srcNodeId.get():
           gossipNodes.add(node)
 
-  if gossipNodes.len >= 8: # use local nodes for gossip
```
Comment (web3-developer) [inline fluffy/network/wire/portal_protocol.nim:1521]: I assume this was hard coded by mistake.

File: fluffy/network/wire/portal_protocol_config.nim [hunk 2]
```diff
@@ -40,17 +40,21 @@ type
     bitsPerHop*: int
     radiusConfig*: RadiusConfig
     disablePoke*: bool
+    maxGossipNodes*: int
 
 const
   defaultRadiusConfig* = RadiusConfig(kind: Dynamic)
   defaultRadiusConfigDesc* = $defaultRadiusConfig.kind
   defaultDisablePoke* = false
+  defaultMaxGossipNodes = 8
```
Comment (web3-developer) [inline fluffy/network/wire/portal_protocol_config.nim:49]: @kdeme Would you like me to lower this to 6 as discussed?

File: fluffy/network/wire/portal_protocol.nim (hunk 1 above)
Comment (kdeme) [reply fluffy/network/wire/portal_protocol.nim:1521]: yes

File: fluffy/network/wire/portal_protocol_config.nim (hunk 2 above)
Comment (kdeme) [reply fluffy/network/wire/portal_protocol_config.nim:49]: Although it will most likely be put lower, lets keep that for the other PR.

**Review by kdeme — APPROVED** [review]

**Pull request merged**